.circleci/config.yml

```diff
@@ -53,8 +53,9 @@ references:
     DO_TF_WORKSPACE: dev
     RELEASE_NAME: codacy
     NAMESPACE: codacy
-    K8S_VERSION: "1.14"
-    NUM_NODES: "5"
+    ENVIRONMENT: dev
```

Comment:
Used for pattern-matching with the templated values-db-<environment>.yaml file created by terraform

Reply:
I don't love this dependency.

Reply:
The main purpose of this is so that the correct values file for the specific environment is picked up in the makefile while installing codacy. 
The db hosts as well as the user and password for that host will change per environment so I don't think we can just use one file for everything. How do you think it would be best to implement this?